src/containers/Map/MapScreen.tsx

```diff
@@ -341,7 +335,7 @@ const formatEventBotsData = (
 			// TODO: fix item images
 			items.push({ ...obj.item, quantity: obj.quantity, bot: bot });
 			itemCount += obj.quantity;
-			itemsSold += obj.sales.numSold;
+			//itemsSold += obj.sales.numSold;
```

Comment:
For some reason this line caused the code to break so I commented it out. Should add to bug report